dasp_signal/src/lib.rs

```diff
@@ -2285,14 +2285,23 @@ impl<S> IntoInterleavedSamples<S>
 where
     S: Signal,
 {
-    /// Yield the next interleaved sample from the inner `Signal`.
+    /// Yield the next interleaved sample from the inner `Signal`. The returned value
```

Comment:
Can you add a blank line between the period and the new sentence? Then the second sentence won't show up in the rustdoc summary, only if you click on it.

Reply:
Done, thanks!

Reply:
I think I didn't change what we expected in the commit I've pushed yesterday. I pushed a new one that achieves the intended effect.